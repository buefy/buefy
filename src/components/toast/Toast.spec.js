import { shallowMount } from '@vue/test-utils'
import BToast from '@components/toast/Toast'

describe('BToast', () => {
    HTMLElement.prototype.insertAdjacentElement = jest.fn()

    it('is called', () => {
        const wrapper = shallowMount(BToast)
        expect(wrapper.name()).toBe('BToast')
        expect(wrapper.isVueInstance()).toBeTruthy()
    })
})
