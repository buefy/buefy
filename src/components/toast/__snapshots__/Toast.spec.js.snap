// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BToast render correctly 1`] = `
<div role="alert" class="toast is-dark is-top" style="" enter-active-class="fadeInDown" leave-active-class="fadeOut">
    <div></div>
</div>
`;
